script.js

```diff
@@ -1,4 +1,62 @@
+var sciSSorsPaperStone = function () {
```

Comment:
Should apply javascript naming convention to function name i.e. camelCase. So should be written as _scissorsPaperStone_. However using _scissorsPaperStone_.is not a really good description of what this function is doing. A better name could be _genComputerHand_.